media/js/firefox/new/experiment-firefox-new-batm-anim.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_batm_anim',
+        variations: {
+            'xv=batmprivate': 33,   // Control: basic page, no machine
```

Comment:
D'oh, now I get it... in the past we've done a "double control" where one variant was the current page, but that was still a redirect. I misunderstood how Traffic Cop was working in those instances.

So, ok... we want people to land at a URL, and 33% stay where they landed, 33% get sent to A and 33% get sent to B... but if where they landed is already an `?xv=` URL Traffic Cop won't know that and will try to add another query string onto it. So... this experiment is impossible without changes to Traffic Cop. 

I'm not sure what to do about this. We probably need to talk to Fabio and figure out what the next step is and if we can rethink the test. I wonder if we'll have to use Optimizely instead 😞